coreapi/main_api.py

```diff
@@ -102,55 +105,54 @@ def nsfwClassifier(request, filename):
     data = np.asarray(img, dtype="float32")
     data = img_standardize(data)
     image_data = data.astype(np.float16, copy=False)
-    SERVER_URL = 'http://localhost:8501/v1/models/nsfw:predict'
-    jsonData = json.dumps({"inputs": [image_data.tolist()]})
+    server_url = 'http://localhost:8501/v1/models/nsfw:predict'
+    jsondata = json.dumps({"inputs": [image_data.tolist()]})
     try:
-        response = requests.post(SERVER_URL, data=jsonData)
+        response = requests.post(server_url, data=jsondata)
     except Exception as e:
         print(e, "\n TensorFlow Serving is not working properly")
         return " "
```

Comment:
make return consitent , if it's null then the output format of the method must be same in each return statement of the method